pkg/controller/machineset/machineset_controller_suite_test.go

```diff
@@ -56,7 +56,7 @@ var (
 func TestMachinesetController(t *testing.T) {
 	RegisterFailHandler(Fail)
 
-	RunSpecs(t, "MachineSet Controller Suite")
+	RunSpecs(t, "Cluster Operator status controller Suite")
```

Comment:
Why has this been changed?

Reply:
I don't remember tbh, but this change came in commit 57805a53802149c35b51ca1e305165739092e202

Reply:
I think we can change the names back, they don't need to be changed

Reply:
Resolved.